### PR TITLE
fix: rustls version conflict when using questdb-rs

### PR DESCRIPTION
**Pull request by jerrinot**
lock ureq to 2.9
a workaround for https://github.com/algesten/ureq/issues/765

### CODE REVIEW

**Review by mtopolnik — APPROVED** [review]

File: corrosion/.github/workflows/gh-pages.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```
Comment (jerrinot) [inline corrosion/.github/workflows/gh-pages.yaml:27]: what's the reason for this update? is it compatible?

File: corrosion/.github/actions/setup_test/action.yaml [hunk 2]
```diff
@@ -138,10 +138,10 @@ runs:
         arch: ${{ steps.arch_flags.outputs.msvc }}
       if: ${{ 'msvc' == steps.determine_abi.outputs.abi }}
     - name: Install CMake
-      uses: corrosion-rs/install-cmake@v2
+      uses: lukka/get-cmake@519de0c7b4812477d74976b2523a9417f552d126
```
Comment (jerrinot) [inline corrosion/.github/actions/setup_test/action.yaml:141]: why this change?

Comment (mtopolnik) [reply corrosion/.github/actions/setup_test/action.yaml:141]: These are changes inside the `corrosion` subtree, which I just copied in full from corrosion github

**Review by amunra — APPROVED** [review]
Looks great!
As an extra bonus I've seen the new updated dependences have fewer transitive dependencies too. Nice!
